geode-core/src/main/java/org/apache/geode/cache/client/internal/ServerDenyList.java

```diff
@@ -56,7 +56,7 @@
   protected final ListenerBroadcaster broadcaster = new ListenerBroadcaster();
 
   // not final for tests.
-  static int THRESHOLD = Integer
+  static final int THRESHOLD = Integer
```

Comment:
This constant is never used outside the class and can also be changed to `private`